pandas/tests/test_expressions.py

```diff
@@ -332,78 +331,81 @@ def testit():
         expr.set_numexpr_threads()
         testit()
 
-    def test_bool_ops_raise_on_arithmetic(self):
+    @pytest.mark.parametrize(
+        "op,name", list(zip(["/", "//", "**"], ["truediv", "floordiv", "pow"]))
```

Comment:
I think would be nicer to follow the `opname, op_str` convention for consistency